tests/e2e/default/tasks/tasks-breadcrumbs-config.js

```diff
@@ -0,0 +1,50 @@
+const oneDay = 24 * 60 * 60 * 1000;
```

Comment:
This line is also in the `due-dates-config.js` file. Can we have just one task config file? Or would it be mixing scenarios?

Reply:
Yeah, indeed, and maybe that line is not the only one ! The thing is `due-dat`e test has its own config, presets and test scenarios, and we also have t`asks-group-config` ; I was wondering if the fusion does worth the confusion, especially if I break some working tests on the way 😕